docker/Dockerfile

```diff
@@ -0,0 +1,130 @@
+FROM ubuntu:16.04
+
+MAINTAINER Daniel Baptista Dias <danielbpdias@gmail.com>
+
+ENV GENSIM_REPOSITORY https://github.com/RaRe-Technologies/gensim/archive
+ENV GENSIM_VERSION master
+
+# Installs python, pip and setup tools (with fixed versions)
+RUN apt-get update \
+    && apt-get install -y \
+    ant=1.9.6-1ubuntu1 \
```

Comment:
Guys, the fixation is needed due best practices with docker. Without these versions fixed the container can broke when a components changes it versions and when someone try to get it from a docker pull.